geth/accounts.go

```diff
@@ -201,6 +221,16 @@ func Logout() error {
 		return fmt.Errorf("%s: %v", ErrWhisperClearIdentitiesFailure, err)
 	}
 
+	swarmService, err := nodeManager.SwarmService()
+	if err != nil {
+		return err
+	}
+
+	err = swarmService.ClearIdentities()
```

Comment:
- again, why can't we stop swarm service directly with `swarmService.Stop()` w/o all those identity-related methods